cli/src/config.rs

```diff
@@ -196,6 +195,21 @@ pub fn get_home_settings_path() -> Result<PathBuf> {
     Ok(config_path)
 }
 
+/// Resolve XDG data directory.
+pub fn data_dir() -> Result<PathBuf> {
+    if let Some(data_dir) = env::var("XDG_DATA_HOME")
+        .ok()
```

Comment:
This is an unnecessary UTF-8 check that can be skipped with `env::var_os`
```suggestion
    if let Some(data_dir) = env::var_os("XDG_DATA_HOME")
```